rclrs_msg_utilities/package.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
```

Comment:
Correct me if I'm wrong, but it seems that `rclrs_msg_utilities` serves a similar purpose as `rclrs_common`, that is, code shared between `rclrs` and downstream packages / generators. Would it make sense to rework `rclrs_common` to include the additions from `rclrs_msg_utilities` ?

Reply:
It cannot, as best as I am able to tell. Every time I try, it introduces a circular build dependency.

Reply:
Oh, sorry, misunderstood your question. So basically what `rclrs_msg_utilities` is is just `rclrs_common` stripped of everything that doesn't have to do with messages. My goal with it is to eventually use it to help with message generation via CMake.

Since I'm not fully sure what the plan is anymore - We were going to switch over to `r2r`, but then I got comments saying that some other people preferred the CMake implementation...